service-commands/scripts/.env/README.md

```diff
@@ -0,0 +1,218 @@
+# Audius Env Migration Manager
```

Comment:
nice docs! maybe useful for non-powerusers - a section with "ELI5 how to migrate an env" step-by-step

Reply:
Thanks! I eliminated some descriptive text geared towards powerusers which should be simpler to understand for all users.